both/collections/round.js

```diff
@@ -30,6 +30,13 @@ _.extend(Round.prototype, {
     assert(this.eventCode);
     return wca.eventByCode[this.eventCode].solveTimeFields;
   },
+  roundCode: function() {
```

Comment:
Good idea. There was no need for these fields to be in the db. I think we had them because we were originally dumbly importing the WCA json, which reminds me: note to self: check that we're still able to import competitions.